resources/chefdk/msi/assets/start-chefdk.ps1

```diff
@@ -0,0 +1,22 @@
+$conemulocation = "$env:programfiles\ConEmu\Conemu64.exe"
```

Comment:
This is ok, but we could do better -- it might not be at `$env:programfiles`. Another option assuming conemu is in the path is to just do a `get-command conemu64.exe` before you do this, and then get the full path from that if it succeeds, otherwise try to find it under programfiles as it is doing.

I'd just ship this for now though, worst case we don't use your conemu and we use the regular terminal instead. It's somewhat less of a tragedy on Win 10 since it now can do line-wrapped selection and fast copy / paste.